.github/workflows/repo2cpe-download.yaml

```diff
@@ -0,0 +1,37 @@
+name: Download Repo2CPE
+on:
+  push:
+    branches:
+    - "yli3/**"
+  schedule:
+  - cron: "0 */3 * * *"
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Authenticate with Google Cloud
+      uses: google-github-actions/auth@v1
+      with:
+        credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
```

Comment:
Can you rename the secret to `GOOGLE_SA_REPOTOCPE_UPLOADER`, that makes it clearer where it is used and hopefully avoids misuse by other workflows :)

Reply:
@daynewlee

Reply:
ok makes sense, thanks for clarifying.